src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 project(System.IO.Compression.Native C)
 
+include(../../../../eng/native/functions.cmake)
+
```

Comment:
```suggestion
```
This is not needed. All cmake scripts in eng/native are available by global includes (otherwise `install_with_stripped_symbols` on line 102 will fail).

Reply:
Removed, thanks!